pkg/internal/controller/controller_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("controller", func() {
 			defer func() {
 				Expect(recover()).To(BeNil())
 			}()
-			ctrl.RecoverPanic = true
+			ctrl.RecoverPanic = ptr(true)
```

Comment:
```suggestion
			ctrl.RecoverPanic = pointer.Bool(true)
```
? 
Just so we keep using the same utilities all around

Reply:
Updated, ptal